14:35:14 - ERR - post - (AttributeError("'dict' object has no attribute 'messages'"),)
14:35:55 - ERR - post - (AttributeError("'dict' object has no attribute 'messages'"),)
14:40:18 - ERR - create_user - (TypeError("'int' object is not subscriptable"),)
14:40:54 - ERR - ExecuteNonQuery - (UniqueViolation('duplicate key value violates unique constraint "usuarios_correo_electronico_key"\nDETAIL:  Key (correo_electronico)=(user@example.com) already exists.\n'),)
14:52:49 - ERR - login_user - (KeyError(0),)
14:59:36 - ERR - post - (KeyError('correo_electronico'),)
14:59:54 - ERR - post - (KeyError('correo_electronico'),)
15:01:23 - ERR - post - (KeyError('correo_electronico'),)
15:01:34 - ERR - post - (KeyError('correo_electronico'),)
15:01:43 - ERR - post - (KeyError('correo_electronico'),)
15:01:54 - ERR - post - (KeyError('correo_electronico'),)
15:02:16 - ERR - post - (KeyError('correo_electronico'),)
15:02:22 - ERR - post - (KeyError('correo_electronico'),)
15:02:24 - ERR - post - (KeyError('correo_electronico'),)
15:02:47 - ERR - post - (KeyError('correo_electronico'),)
